index.js

```diff
@@ -163,7 +163,7 @@ program
       })
       .then(() => {
         const watcher = chokidar.watch(rootFolder, {
-          ignored: [outputFolder, /(^|[/\\])\../],
+          ignored: [outputFolder, /(^|[/\\])\../, '/**/*.md__jb_tmp__'],
```

Comment:
@nicholaschuayunzhi Can test this? I don't get this files when I use intellij or webstorm

Reply:
@rachx, I'm still getting changes recorded.
![image](https://user-images.githubusercontent.com/22221132/37072460-aa98b2de-21fc-11e8-8f60-0e9950bbde7a.png)

Also do consider ignoring html tmp files
![image](https://user-images.githubusercontent.com/22221132/37072495-dc95482e-21fc-11e8-9415-1abe958944cf.png)

Reply:
this worked for me:
```js
ignored: [outputFolder, /(^|[/\\])\../, '*.md__jb_tmp__'],
```